### PR TITLE
added Kimai v2 announcement

### LINKED ISSUE #259: Technology switch?
I met Torsten (the kimai founder) 5 years ago and we talked about ExtJS to use for frontend. We did decide to decide it later.
Now it's 2013 and Kimai is based on technologies 10 years old.

My suggest for `Kimai2`:
- backend based on [Symfony2](https://github.com/symfony/symfony)
- frontend based on [Bootstrap](https://github.com/twbs/bootstrap) or Sencha [ExtJS](http://www.sencha.com/products/extjs/)/[Touch](http://www.sencha.com/products/touch/)

### PR DESCRIPTION
**Pull request by kevinpapst**
Closes #259

See https://github.com/kimai/www.kimai.org/pull/19

### CODE REVIEW

**Comment by kevinpapst** [comment]
@simonschaufi any comment?

File: README.md [hunk 1]
```diff
@@ -18,6 +18,12 @@ You can get more information about this time-tracking software:
 * at our [website](https://www.kimai.org)
 * in the [documentation](https://www.kimai.org/documentation/)
 
+## Announcement Kimai v2 (June 2018)
+
+We are thrilled to announce  that we are working on the **next version of Kimai v2**.
```
Comment (simonschaufi) [inline README.md]: the next major version of Kimai

File: README.md [hunk 2]
```diff
@@ -18,6 +18,12 @@ You can get more information about this time-tracking software:
...
+
+Please [read our blog posting](https://www.kimai.org/blog/2018/kimai-v2-announcement/) to find out more.
```
Comment (simonschaufi) [inline README.md]: ~~Please~~

**Review by simonschaufi — CHANGES_REQUESTED** [review]

**Comment by kevinpapst** [comment]
Check again, I don't think Scrutinizer or Travis will have any comment ;-)

**Review by simonschaufi — APPROVED** [review]

**Comment by kevinpapst** [comment]
Could you cherry pick that into master? Or how do we get it to the front page of the repo?

**Comment by simonschaufi** [comment]
I just merge it. When there is no additional commit in master branch, it won't create a merge commit.

**Comment by kevinpapst** [comment]
Thanks a lot 👍  

Yeah, I accidentally merged a PR in master this week I guess 👎 just saw it after I merged... sorry!

**Comment by simonschaufi** [comment]
Don't worry, no big deal. As nothing crashed...

**Comment by simonschaufi** [comment]
More important is the Milestone information as this will be included in the release notes.

**Pull request merged**